tests/unit/test_plotting/test_quick_plot.py

```diff
@@ -429,6 +429,7 @@ def test_plot_2plus1D_spm(self):
         quick_plot.slider_update(1)
 
         # check 2D (y,z space) variables update properly for different time units
+        # Note: these should be the transpose of the entries in the processed variable
```

Comment:
I am confused: the comment says it should be the transpose but below it is not transposed

Reply:
In the test we do, e.g., `np.testing.assert_array_almost_equal(qp_data.T, phi_n[:, :, 0])` so that `qp_data` is transposed